docs/getting-started.adoc

```diff
@@ -54,13 +54,29 @@ $ odo push
 ----
 +
 Your component is now deployed to OpenShift.
-. To access the component, create an OpenShift route and view your
-deployed application using the generated URL:
+. To access the component, create an URL:
 +
 ----
-$ odo url create --open
+$ odo url create --port 8080
 ----
-
++
+. Previous command will make entry in config file, to create URL on cluster:
++
```

Comment:
There seems to be some mix up here. Has an explanation for the previous step been mixed up with the next instruction step. In that case we will have to split it as mentioned here and in the next command.
Suggest: This also adds an entry in the configuration file. 
Please remove the period (it creates a numbered step) at the beginning of the sentence, as it is not an user action/next step. It simply explains what happens in the previous step.